### DISCUSSION TITLE
Website Release - 0.1.2 - 2022-09-11

### DISCUSSION THREAD

**Discussion by WengerK** [Games-of-Switzerland/swissgamesgarden#119] [Announcements]
### Security
- update library react-query 3.34.14 => 3.39.2

### Fixed
- fix the pager - Games-of-Switzerland/swissgamesgarden#82

[0.1.2]: https://github.com/Games-of-Switzerland/gos-website/compare/0.1.1...0.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Games-of-Switzerland/swissgamesgarden/releases/tag/0.1.2'>0.1.2</a>.</em>

**Comment by yrochat** [comment]
Merci beaucoup pour ces deux releases consécutives !
